docs/resources/profile_urlfilter.md

```diff
@@ -3,23 +3,9 @@ Resource for creating and managing a URL Filter (URL) Profile
 
 ## Example Usage
 ```hcl
-resource "valtix_profile_urlfilter" "url1" {
+resource "ciscomcd_profile_urlfilter" "url1" {
```

Comment:
shouldnt this be valtix_profile_urlfilter?

Reply:
Updated to correct